frontend/src/features/search/BigSearchBox.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import classNames from 'classnames'
+
+import Icon from '../../chrome/Icon'
+
+interface BigSearchBoxProps {
+  onSubmit: (q: string) => void
+  value: string | undefined
+  className?: string
+}
+
+const BigSearchBox: React.FC<BigSearchBoxProps> = ({ onSubmit, value, className }) => {
```

Comment:
give value a default empty string to ensure value will always be a string type. remove `|| ''` from line 35